### DISCUSSION TITLE
My account was suspend for 40 days with no response, they still charge my copilot fee every month

### DISCUSSION THREAD

**Discussion by CraftBotics** [community/community#120280] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Ticket ID: 2719633

No reason, no email. no response. such a terrible company.

**Comment by fahadshahbaz** [comment]
I also found myself in a situation recently where my account was suspended without any email or message. I raised a support ticket, explained my problem, and it took 2 to 3 weeks for my account to be unsuspended. Be patient, I know it's strange and overwhelming but thats the only thing you can do .

**Comment by CraftBotics** [comment]
No one month have passed, no update, i still can not log in. I have no way to cancel copilot subscription. they never reply my email. any one know a better way to find somebody take care of this

**Comment by fahadshahbaz** [comment]
@CraftBotics Are you having trouble logging in? If you're unable to log in, you won't be able to raise a ticket for assistance. Perhaps you can reach out to support from another account for help with accessing your original account.

**Reply by CraftBotics** [reply]
No I can not login, let me try it out your way. use this account to do it. thank you very much
